src/compiler/checker.ts

```diff
@@ -2763,6 +2763,14 @@ namespace ts {
                         }
                         return hasAccessibleDeclarations;
                     }
+                    else {
+                        if (some(symbol.declarations, hasExternalModuleSymbol)) {
+                            // Any meaning of a module symbol is always accessible via an `import` type
+                            return {
+                                accessibility: SymbolAccessibility.Accessible

```

Comment:
there are a set of error messages that should not used now, e.g. `* from external module blah and can not be named` errors.

Reply:
_Technically those are still possible_:
```ts
// file1:
export function f() {
  interface Local { x }
  const y: Local;
  return y;
}
// file2:
import { f } from "./file1"
export default f();
```

Reply:
i see..